test.js

```diff
@@ -132,26 +132,49 @@ describe('Persistence', (done) => {
 
   it('Record a vote to a restarted session', (done) => {
     chai.request(app)
-    .post('/actions')
-    .send({
-      payload: JSON.stringify({
-        channel: { id: 1 },
-        actions: [{ value: 'Medium' }],
-        user: { name: 'User 2'},
-        original_message: { text: 'Feature 1' }
+      .post('/actions')
+      .send({
+        payload: JSON.stringify({
+          channel: { id: 1 },
+          actions: [{ value: 'Medium' }],
+          user: { name: 'User 2' },
+          original_message: { text: 'Feature 1' }
+        })
+      })
+      .end((err, res) => {
+        res.should.have.status(200)
+        res.should.be.json
+        res.body.attachments[0].text.should.have.string('2 vote')
+        res.body.attachments[0].text.should.have.string('User 1')
+        res.body.attachments[0].text.should.have.string('User 2')
+
+        done()
+      })
+  })
+
+})
+
+describe('Minor Fixes', (done) => {
+
+  it('Dificulty -> Difficulty spelling fix', (done) => {
+    chai.request(app)
+      .post('/commands')
+      .send({
+        channel_id: '18_fix_spelling_issue channel_id',
+        text: '18_fix_spelling_issue text'
+      })
+      .end((err, res) => {
+        res.should.have.status(200)
+        res.should.be.json
+
+        const responseText = res.body.attachments[0].text
+        responseText.should.have.string('Please choose a difficulty')
```

Comment:
I would suggest adding this assertion to the existing tests rather than creating new ones for fixes.

[Start a voting session](https://github.com/AgileVentures/asyncvoter-slack-command/pull/20/files#diff-1dd241c4cd3fd1dd89c570cee98b79ddR59) seems to be a good place for checking for user indications.